x-pack/plugins/fleet/common/types/rest_spec/epm.ts

```diff
@@ -59,6 +59,7 @@ export interface InstalledPackage {
   dataStreams: Array<{
     name: string;
     title: string;
+    humanReadableTitle?: string;
```

Comment:
**nit**: My bad on the poor issue title, what do you think about displayName as the property name? It would align with other places where we use similar naming.